lib/clients/json/jsonClient.ts

```
@@ -0,0 +1,107 @@
+            })
+        })
+        return Array.from(tagsMap.values())
+    }
+
+    return {
+        getAllPosts: async () =>
+            [...posts].filter(Boolean).sort((a, b) => {
+                const dateA = new Date(a.date).getTime()
+                const dateB = new Date(b.date).getTime()
+
+                if (isNaN(dateA) && isNaN(dateB)) return 0
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)